docs/using.md

```diff
@@ -209,6 +209,39 @@ Creation of a `PipelineRun` will trigger the creation of
 
 See [the example PipelineRun](../examples/runs/pipeline-run.yaml).
 
+### Using a `ServiceAccount`
```

Comment:
nit: When `` is used for a word in header markdown renders them in weird fashion. Probably try 

```
### Using ServiceAccount
```

or 

```
### Using a ServiceAccount
```

Reply:
:upside_down_face: